examples/02_model_collaborative_filtering/multi_vae_deep_dive.ipynb

```diff
@@ -0,0 +1,1829 @@
+{
```

Comment:
<p>historicity -&gt; history</p><p> </p><p>It is a bit unclear where NDCG etc. are computed according to the paper (only the _te items are ranked, right?). Is this what <code>map_back_sparse()</code> doing or does it include the _tr items too? It would be good to clarify the protocol in the text.&nbsp;</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
_te dataset contains ratings and _tr are click matrices.